spec/asyncapi.md

```diff
@@ -135,11 +135,14 @@ A sender is a type of application, that is sending [messages](#definitionsMessag
 ### <a name="definitionsReceiver"></a>Receiver
 A receiver is a type of application that is receiving [messages](#definitionsMessage) from [channels](#definitionsChannel). A receiver MAY receive from multiple channels depending on the [server](#definitionsServer), protocol, and the use-case pattern. A receiver MAY forward a received message further without changing it. A receiver MAY act as a consumer and react to the message. A receiver MAY act as a processor that, for example, aggregates multiple messages in one and forwards them.
 
+### <a name="definitionsOperation"></a>Operation
+An operation describes a specific action an [application](#definitionsApplication) can take to interact with the message-driven API. Operations are performed by sending (or receiving) [messages](#definitionsMessage) to (or from) [channels](#definitionsChannel).
```

Comment:
```suggestion
An operation describes a specific action an [application](#definitionsApplication) can take to interact with the [server](#definitionsServer). Operations are performed by sending (or receiving) [messages](#definitionsMessage) to (or from) [channels](#definitionsChannel).
```

Reply:
An operation in spec 3 is something like `userLogin`, `orderGoods`, or `sendPaymentInfo`. All of these make sense when viewed as an operation of the message-driven API defined in an asyncapi document but not when viewed as an operation on, e.g., a kafka server that is only a part of the API implementation with the role of relaying and storing messages.
The old 2.x operations were operations like `send` and `receive` that were specifically tied to a channel/server. I am against associating the new spec 3 operation with servers, as this would make this difference harder to spot at a glance.

Reply:
Would that work for you instead?

```suggestion
An operation describes a specific action an [application](#definitionsApplication) can take to interact with a [channel](#definitionsChannel). Operations are performed by sending (or receiving) [messages](#definitionsMessage) to (or from) [channels](#definitionsChannel).
```

I don't want to even mention the words message-driven and API because they're not defined anywhere, and their meaning is already bloated. I'm actually getting rid of the word API as much as possible because in the message-driven world, people don't perceive it as an API (that's often associated with HTTP).

Also, AsyncAPI doesn't only serve the purpose for message-driven systems. Note that we also support HTTP interactions (even more with v3). Even though, theoretically, HTTP APIs could be broken down to message exchange interactions.

Reply:
Alright. For some reason I thought we already removed all mentions of message-driven API but I can see we didn't. Good to go then 👍